pgcli/packages/sqlcompletion.py

```diff
@@ -129,28 +149,27 @@ def suggest_special(text):
     if cmd[1:] == 'd':
         # \d can descibe tables or views
         if schema:
-            return [{'type': 'table', 'schema': schema},
-                    {'type': 'view', 'schema': schema}]
+            return (Table(schema=schema),
+                    View(schema=schema),)
         else:
-            return [{'type': 'schema'},
-                    {'type': 'table', 'schema': []},
-                    {'type': 'view', 'schema': []}]
+            return (Schema(),
+                    Table(schema=None),
+                    View(schema=None),)
     elif cmd[1:] in ('dt', 'dv', 'df', 'dT'):
-        rel_type = {'dt': 'table',
-                    'dv': 'view',
-                    'df': 'function',
-                    'dT': 'datatype',
+        rel_type = {'dt': Table,
+                    'dv': View,
+                    'df': Function,
+                    'dT': Datatype,
                     }[cmd[1:]]
         if schema:
-            return [{'type': rel_type, 'schema': schema}]
+            return rel_type(schema=schema),
         else:
-            return [{'type': 'schema'},
-                    {'type': rel_type, 'schema': []}]
+            return Schema(), rel_type(schema=None)
 
     if cmd in ['\\n', '\\ns', '\\nd']:
-        return [{'type': 'namedquery'}]
+        return NamedQuery(),
 
-    return [{'type': 'keyword'}, {'type': 'special'}]
+    return Keyword(), Special()
```

Comment:
Surrounding parens here as well. I know it's not a single value tuple but having the parens emphasizes that it's a tuple.